tests/containers/bci_test.pm

```diff
@@ -88,7 +88,7 @@ sub run {
     reset_container_network_if_needed($engine);
 
     record_info('Run', "Starting the tests for the following environments:\n$test_envs");
-    assert_script_run("cd /root/BCI-tests");
+    assert_script_run("cd /root/BCI-tests && git fetch && git reset --hard origin");
```

Comment:
interesting, I would have just done a `git pull`, but this looks better :)

Reply:
This also works if there are changes that are not fast-forward.